servers/rendering_server.h

```diff
@@ -491,8 +491,8 @@ class RenderingServer : public Object {
 		SHADOW_QUALITY_MAX
 	};
 
-	virtual void shadows_quality_set(ShadowQuality p_quality) = 0;
-	virtual void directional_shadow_quality_set(ShadowQuality p_quality) = 0;
+	virtual void positional_shadow_filter_quality_set(ShadowQuality p_quality) = 0;
+	virtual void directional_shadow_filter_quality_set(ShadowQuality p_quality) = 0;
```

Comment:
I wonder why these methods are in Yoda speak instead of following our usual convention of `<category>_set_<thing>`. `directional_shadow_set_filter_quality` seems better and more consistent with other RS methods (e.g. `viewport_set_positional_shadow_atlas_size` modified below).

Reply:
Our convention for most of the RenderingServer is to do ``resource_name_set_property_name()`` e.g. environment is:
https://github.com/godotengine/godot/blob/8c8eea9428d094c3f2cc6731e3e2cbd84a5a316b/servers/rendering/renderer_rd/renderer_scene_render_rd.h#L1061-L1068

This includes quality settings e.g. ``sub_surface_scattering_set_quality()``, ``environment_set_ssao_quality()``

We should just rename to ``positional_shadow_filter_set_quality`` and ``directional_shadow_filter_set_quality``